docs/documentation/tutorial.md

```diff
@@ -176,9 +176,11 @@ Let’s also add a script to `package.json` to make serving the web app simpler:
 {
   "scripts": {
     "start": "parcel index.html --open"
-  }
+  },
+  "browserslist": "> 0.4% and not ie 11",
```

Comment:
I'm not sure if the tutorial should include the trailing `,` since it depends on the surrounding json